From e9d2cca0f83075a946fef81012cf60bb73fee73b Mon Sep 17 00:00:00 2001
From: Ali Abdallah <ali.abdallah@suse.com>
Date: Thu, 27 Feb 2025 23:24:25 +0100
Subject: [PATCH 1002/1005] OnCalendar format fix of mdcheck_start.timer

This patch includes the fix of the OnCalendar format, changing the format of
mdcheck_start.timer [Timer] section,
from OnCalendar=Sun *-*-1..7 1:00:00
to   OnCalendar=Sun *-*-* 1:00:00

Signed-off-by:  Ali Abdallah <ali.abdallah@suse.com>
Acked-by: Coly Li <colyli@suse.de>
---
 systemd/mdcheck_start.timer | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/systemd/mdcheck_start.timer b/systemd/mdcheck_start.timer
index 9e7e02a..ba15ef5 100644
--- a/systemd/mdcheck_start.timer
+++ b/systemd/mdcheck_start.timer
@@ -9,7 +9,7 @@
 Description=MD array scrubbing
 
 [Timer]
-OnCalendar=Sun *-*-1..7 1:00:00
+OnCalendar=Sun *-*-* 1:00:00
 
 [Install]
 WantedBy= mdmonitor.service
-- 
2.48.1

